gameSource/minitech.cpp

```diff
@@ -0,0 +1,1355 @@
+
+#include <windows.h>
```

Comment:
Including windows.h removes linux compatibility, what's the reason for including it?